packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -149,11 +151,12 @@ curl -v -X POST \
     "validationStatus": "VERIFIED",
     "dnsRecords": [
         {
-            "fqdn": "_oktaverification.login.example.com",
+          "recordType": "TXT",
+          "fqdn": "_oktaverification.login.example.com",
             "values": [
                 "79496f234c814638b1cc44f51a782781"
             ],
-            "recordType": "TXT"
+          "expiration": "2021-11-23T02:04:28.000Z"
```

Comment:
same as above, since certificateSourceType in the response is "Manual". 

Maybe we can add one more sample response for okta-managed..?  As the response will be bit different =>  "fqdn" field will be acmeverification.* with an "expiration" value and certificate metadata section will also be included in response.